### PR TITLE
Add StringBuilder

### PR DESCRIPTION
**Pull request by alexfilimon**
Add class `StringBuilder` to simplify creating complicated `NSAttributedString` with different attributes for different parts of text. 

There are two types of attributes: local and global. In initialisation time you add global. With `add(text: String, attributes: [StringAttribute])` you add local styles.

Render into `attributeString` perfoms only in time of acmes to `value` property. 

Syntax:
```swift
let attributedString = StringBuilder(
    attributes: [
      .font(FontFamily.MyFont.regular.font(size: 24)),
      .foregroundColor(Color.Main.blackText)
    ]
  )
  .add(text: "12", with: [
    .font(FontFamily.MyFont.regular.font(size: 36)),
    .foregroundColor(Color.Main.redError)
  ])
  .addSpace()
  .add(text: "месяцев")
  .addLineBreak()
  .add(text: "green", with: [
    .foregroundColor(.green)
  ])
  .value
```

### CODE REVIEW

File: Utils/Utils/String/StringBuilder.swift [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+//  StringBuilder.swift
+//  Utils
+//
+//  Created by Alexander Filimonov on 28/06/2019.
+//  Copyright © 2019 Surf. All rights reserved.
+//
+
+import Foundation
+
+public class StringBuilder {
```
Comment (avsmirnov567) [inline Utils/Utils/String/StringBuilder.swift]: I guess, you can leave small documenting comment about a purpose of the class.

File: Utils/Utils/String/StringBuilder.swift [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    }
+
+    // MARK: - Private properties
+
+    private var parts: [StringPart] = []
+    private var globalAttributes: [StringAttribute] = []
+
+    // MARK: - Initialization
+
+    public init(attributes: [StringAttribute] = []) {
```
Comment (avsmirnov567) [inline Utils/Utils/String/StringBuilder.swift]: Maybe "globalAttributes" will be more clear?

File: Utils/Utils/String/StringBuilder.swift (hunk 1 above)
Comment (alexfilimon) [reply Utils/Utils/String/StringBuilder.swift]: Yes, made it

File: Utils/Utils/String/StringBuilder.swift (hunk 2 above)
Comment (alexfilimon) [reply Utils/Utils/String/StringBuilder.swift]: Yes, made it

**Pull request merged**